Caddyfile

```diff
@@ -1,10 +1,10 @@
-{$CADDY_URL}
-root * /usr/share/caddy
-route /{$CADDY_API_ENDPOINT}/* {
-	uri strip_prefix /API_END_POINT
-	reverse_proxy {$CADDY_PROXY_URL}
+{$CADDY_URL} {
+    handle_path /{$CADDY_API_ENDPOINT}* {
```

Comment:
Also, the api endpoint path isn't needed in this Caddyfile since it's not acting as the reverse proxy, so these API paths will have to be removed.

Reply:
To basically respond to all is just that this is the setup I am currently running with no 404s. It's covering the whole stack with a single webserver basically.

Reply:
Just so I'm getting this right you mean "one" webserver being the main reverse proxy (in your case I assume Caddy?) that would be setup independent from all projects (excluding the new docker-compose file)? If so, that's good to know! Then I am willing to accept the changes but please make sure to not include the reverse proxy directives in this file as it is not needed inside the container, it should be pushed up one level :) I think you should be able to remove all variables in this file as well, as I've done in the branch you are merging into. If you just rebase onto that and make the merge conflict fixes I should be able to merge both in. Please note that I'm only doing a light file, as a full file should not be needed, and I would like to move away from that as well.

Reply:
The light file is my choice all the way, everytime. The levels was what I was avoiding. It's this weird hybrid setup of three containers: mariadb, g5api and g5v that provides the dist files and a webserver+reverse proxy all in that third one. I did some experiments assemling it into a compose file, but not successfully, yet. You can think of it as a very monolithic deployment. But very easy for basic setups.

Reply:
I mean, if you want an easy setup you should be able to use this one just fine, and it uses Caddy as the reverse proxy https://github.com/PhlexPlexico/G5API/blob/master/docker-compose.yml 

But yeah, I think we're on the same page then. It's just cutting down to semantics and I guess if someone wants to roll everything with Caddy, I suppose it should be allowed (even though it doesn't really make any difference at this level). Once resolved, I can merge everything in. Just make sure the Caddyfile stays similar to the changes that were recently made (i.e. no build variables needed) and it should be good to merge :)